source/Preferences.cpp

```diff
@@ -37,6 +37,9 @@ namespace {
 	int zoomIndex = 4;
 	constexpr double VOLUME_SCALE = .25;
 
+	int screenModeIndex = 0;
+	const vector<string> SCREEN_MODE_SETTINGS = {"windowed", "fullscreen"};
```

Comment:
Do not change this, but...
Just so  you know, you did not actually need to do this since it is a bool, you could have just held it in one Preference::("fullscreen") (that already exists). We only actually need to do this when there are more than 2 options.

That said I much prefer it being done this way, to allow for easily adding windowed fullscreen later on.

Reply:
Yep, hopefully we can just slot the borderless windowed mode right into there.